tests/MongoDB.Driver.Tests/Specifications/command-monitoring/CommandMonitoringTestRunner.cs

```diff
@@ -318,78 +358,25 @@ private BsonDocument MassageReply(string commandName, BsonDocument reply, BsonDo
             return massagedReply;
         }
 
-        private BsonValue DeepCopy(BsonValue value)
+        // nested types
+        private class TestCaseFactory : JsonDrivenTestCaseFactory
         {
-            if (value.BsonType == BsonType.Document)
-            {
-                var document = new BsonDocument();
-                foreach (var element in (BsonDocument)value)
-                {
-                    document.Add(element.Name, DeepCopy(element.Value));
-                }
-
-                return document;
-            }
-            else if (value.BsonType == BsonType.Array)
-            {
-                var array = new BsonArray();
-                foreach (var element in (BsonArray)value)
-                {
-                    array.Add(DeepCopy(element));
-                }
-                return array;
-            }
+            // protected properties
+            protected override string PathPrefix => "MongoDB.Driver.Tests.Specifications.command_monitoring.tests.";
 
-            return value;
-        }
```

Comment:
The private method `DeepCopy` is substituted everywhere in the class with `BsonDocument.DeepClone()` which does the same thing.